transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -153,7 +150,10 @@ public ChannelHandlerContext fireUserEventTriggered(Object event) {
     @Override
     public ChannelHandlerContext fireChannelRead(Object msg) {
         AbstractChannelHandlerContext next = findContextInbound();
-        ReferenceCountUtil.touch(msg, next);
+        if (ResourceLeakDetector.ENABLED) {
```

Comment:
@normanmaurer it's not supposed to be wooo 5% cpu boost. I just want it out of the hot spots.